frontend/components/TeamSection/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Heading, GradientText, HEADING_OPTIONS } from '../shared/Heading';
+import PersonCard from '../person-card';
+
+const TeamGroupSection = ({ data }) => {
+  const { title, cardsData } = data;
+
+  const membersData = cardsData.map(cardData => {
+    return (
+      <div className="w-1/2 mx-auto border border-c1000">
+        <PersonCard
+          className="border border-c1000"
```

Comment:
components can't take className like other html element it needs to be a prop in the component. but since you added the border to the top div you don't need this class.